supportedBackends/tes/src/test/scala/cromwell/backend/impl/tes/TesRuntimeAttributesSpec.scala

```diff
@@ -1,7 +1,7 @@
 package cromwell.backend.impl.tes
 
 import cromwell.backend.validation.ContinueOnReturnCodeSet
-import cromwell.backend.{BackendConfigurationDescriptor, MemorySize, RuntimeAttributeDefinition, TestConfig}
+import cromwell.backend.{BackendConfigurationDescriptor, RuntimeAttributeDefinition, TestConfig}
```

Comment:
I think you need to import wom.format.MemorySize

Reply:
Oops! Added.